### ISSUE TITLE
ComboBox Options don't respect title

### ISSUE THREAD

**Issue by clayplumridge** [microsoft/fluentui#18087]
### Environment Information

- **Package version(s)**: 1.170.0
- **Browser and OS versions**: Chrome 90.0.4430.93

### Please provide a reproduction of the bug in a codepen:

https://codepen.io/clayplumridge/pen/wvJvbYv

#### Actual behavior:

When providing alternate `title`s to ComboBoxOptions, the ComboBox does not respect them and instead uses the Text. This includes using empty strings to attempt to remove the tooltip.

#### Expected behavior:

Title and Tooltip content should respect the passed-in `title`, and should be removed if `title` is an empty string.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Low

**Comment by ecraig12345** [comment]
This was previously filed under #16913, but since you provided a repro (thanks!) and the older issue didn't have one, I closed the older issue in favor of this one. @TristanWatanabe This one seems worth looking into and fixing sometime soon since it affects accessibility.

**Comment by TristanWatanabe** [comment]
Hey @clayplumridge - if you pass in an empty string as `title` , no tooltip is shown :)
